create_box/main.py

```diff
@@ -2,8 +2,15 @@
 
 
 def create_box(height, width, character):
-    pass
+    if height>=1 and width>=1:
+        return (character*width + '\n')*height
+    else:
+        return 'must be greater than or equal to 1'
 
+# Extra Credit.
 
-if __name__ == '__main__':
-    create_box(3, 4, '*')
+def create_border_box(height, width, character):
+    if height>=1 and width>=1:
+        r = (character*width + '\n')
+        c = character+' '*(width-2)+character+'\n'
+        return r+c*(height-2)+r
```

Comment:
I understand what you're trying to do here, but your code would be more readable with better variable names. Something like `row` and `col` is much more descriptive than `r` and `c`.